src/config/pragmas/tables.js

```diff
@@ -1,42 +1,59 @@
-module.exports = function configureTables ({ arc }) {
+let is = require('../../lib/is')
+let validate = require('./validate')
+
+module.exports = function configureTables ({ arc, errors }) {
   if (!arc.tables || !arc.tables.length) return null
 
-  let tables = arc.tables.map(t => {
-    if (typeof t === 'object' && !Array.isArray(t)) {
-      let name = Object.keys(t)[0]
-      let table = {
-        name,
-        partitionKey: null,
-        partitionKeyType: null,
-        sortKey: null,
-        sortKeyType: null,
-        stream: null,
-        ttl: null,
-        encrypt: null,
-        PointInTimeRecovery: null,
-      }
-      Object.entries(t[name]).forEach(([ key, value ]) => {
+  let isPrimaryKey = val => val.startsWith('*String') || val.startsWith('*Number')
+  let isSortKey = val => val.startsWith('**String') || val.startsWith('**Number')
```

Comment:
Can we lift these two functions into the global scope so that we don't re-define them on every invocation of this module (teeny tiny perf boost)? Related question: do these belong in the `lib/is` module?

Reply:
Moved to `lib/is`!